code/datums/wounds/robotic/robotic_blunt.dm

```diff
@@ -0,0 +1,1017 @@
+	return (potential_treater.tool_behaviour == TOOL_SCREWDRIVER || potential_treater.tool_behaviour == TOOL_WRENCH || istype(potential_treater, /obj/item/stack/medical/bone_gel))
+
+// Only really relevant for T2/T3 wounds: The primary way to secure internals, best done by someone else
+/datum/wound/blunt/robotic/proc/secure_internals_normally(obj/item/securing_item, mob/user)
+	if (!securing_item.tool_start_check())
+		return TRUE
+
+	var/chance = 10
+	var/delay_mult = 1
+
+	if (user == victim)
+		chance *= 0.2
+		delay_mult *= 2
+
+	if (HAS_TRAIT(user, TRAIT_KNOW_ROBO_WIRES))
+		chance *= 15 // almost guaranteed if its not self surgery
+		delay_mult *= 0.5
+	if (HAS_TRAIT(user, TRAIT_KNOW_ENGI_WIRES))
+		chance *= 8
+		delay_mult *= 0.85
+	if (HAS_TRAIT(user, TRAIT_DIAGNOSTIC_HUD))
+		chance *= 3
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		chance *= 2
+		delay_mult *= 0.8
+
+	var/confused = (chance < 25) // generate chance beforehand, so we can use this var
+
+	var/their_or_other = (user == victim ? "their" : "[user]'s")
+	var/your_or_other = (user == victim ? "your" : "[user]'s")
+	if (user)
+		user.visible_message(span_notice("[user] begins the delicate operation of securing the internals of [their_or_other] [limb.plaintext_zone] internals..."))
+	if (confused)
+		to_chat(user, span_warning("You are confused by the layout of [your_or_other] [limb.plaintext_zone]! Perhaps a roboticist, an engineer, or a diagnostic HUD would help?"))
+
+	if (!securing_item.use_tool(target = victim, user = user, delay = (10 SECONDS * delay_mult), volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+
+	if (prob(chance))
+		if (user)
+			user.visible_message(span_green("[user] finishes securing the internals of [their_or_other] [limb.plaintext_zone]!"))
+		to_chat(victim, span_green("Your [limb.plaintext_zone]'s internals are now secure! Your next step is to weld/cauterize it."))
+		ready_to_secure_internals = FALSE
+		ready_to_ghetto_weld = TRUE
+	else
+		if (user)
+			user.visible_message(span_warning("[user] screws up and accidentally damages [their_or_other] [limb.plaintext_zone]!"))
+		limb.receive_damage(brute = 5, damage_source = securing_item, wound_bonus = CANT_WOUND)
+
+	return TRUE
+
+// If we dont want to use a wrench/screwdriver, we can just use bone gel
+/datum/wound/blunt/robotic/proc/apply_gel(obj/item/stack/medical/bone_gel/gel, mob/user)
+
+	if (gelled)
+		to_chat(user, span_warning("[user == victim ? "Your" : "[victim]'s"] [limb.plaintext_zone] is already filled with bone gel!"))
+		return TRUE
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	user.visible_message(span_danger("[user] begins hastily applying [gel] to [victim]'s' [limb.plaintext_zone]..."), span_warning("You begin hastily applying [gel] to [user == victim ? "your" : "[victim]'s"] [limb.plaintext_zone], disregarding the bold \"ONLY USE WITH ORGANICS\" label..."))
```

Comment:
Ok, I know this is supposedly a method to inform the user this bone gels are a subpar solution, but I don't see any bold "ONLY USE WITH ORGANICS" label in either the desc of bone gels nor while examining them, and that's quite inconsistent.